packages/core/upload/admin/src/components/FolderList/tests/FolderList.test.js

```diff
@@ -48,80 +21,8 @@ const ComponentFixture = (props = { onSelectFolder: jest.fn(), onEditFolder: jes
 const setup = props => render(<ComponentFixture {...props} />);
 
 describe('FolderList', () => {
-  test('renders', () => {
-    const { container } = setup({
-      title: 'List title',
-      folders: FIXTURE_FOLDERS,
-      selectedFolders: [],
-    });
-    expect(container).toMatchSnapshot();
-  });
-
-  test('renders with size=small', () => {
-    const { container } = setup({
-      folders: FIXTURE_FOLDERS,
-      size: 'S',
-    });
-    expect(container).toMatchSnapshot();
-  });
-
-  test('does select selected folders', () => {
-    const { container } = setup({
-      title: 'List title with selected items',
-      folders: FIXTURE_FOLDERS,
-      selectedFolders: FIXTURE_FOLDERS,
-    });
+  it('renders and match snapshots', () => {
+    const { container } = setup();
     expect(container).toMatchSnapshot();
   });
-
-  test('does call onChangeFolder', () => {
-    const spy = jest.fn();
-    const { getByText } = setup({
-      folders: FIXTURE_FOLDERS,
-      onChangeFolder: spy,
-    });
-
-    const folder1 = getByText(FIXTURE_FOLDERS[0].name);
-
-    act(() => {
-      fireEvent.click(folder1);
-    });
-
-    expect(spy).toHaveBeenCalledWith(1);
-  });
-
-  test('does call onEditFolder', () => {
-    const spy = jest.fn();
-    const { container } = setup({
-      folders: FIXTURE_FOLDERS,
-      onEditFolder: spy,
-    });
-
-    const folder1 = container.querySelector('button[aria-label="Edit folder"]');
-
-    act(() => {
-      fireEvent.click(folder1);
-    });
-
-    expect(spy).toHaveBeenCalledWith(FIXTURE_FOLDERS[0]);
-  });
-
-  test('does call onSelectFolder', () => {
```

Comment:
Could you add these tests back? The test coverage goes down quite a bit and it would be nice, if we had tests for these interactions ...

Reply:
discussing this elsewhere - this component doesn't handle those props anymore